api/proto/teleport/legacy/types/types.proto

```diff
@@ -6946,6 +6946,9 @@ message PluginJamfSettings {
 message PluginOktaSettings {
   option (gogoproto.equal) = true;
 
+  reserved 6;
+  reserved "enable_system_log_export";
+
```

Comment:
I'd suggest keeping the reserved field in order (after `credentials_info = 5;` in this case). It makes things slightly easier when introducing a new field.